src/screens/signInScreen/signInScreen.jsx

```diff
@@ -73,7 +72,7 @@ function SignInScreen({ signIn, navigation }) {
         title="Sign In"
         onPress={onSignIn}
       />
-      <View style={styles.signupContainer}>
+      {/* <View style={styles.signupContainer}>
```

Comment:
why is it commented?

Reply:
Igor asked to temporarily remove sign up functionality.